npm/react/examples/snapshots/cypress.config.js

```diff
@@ -3,16 +3,16 @@ module.exports = {
   'fixturesFolder': false,
   'viewportWidth': 500,
   'viewportHeight': 500,
-  'ignoreSpecPattern': [
-    '**/__snapshots__/*',
-    '**/__image_snapshots__/*',
-  ],
   'env': {
     'cypress-plugin-snapshots': {
       'prettier': true,
     },
   },
   'component': {
+    'ignoreSpecPattern': [
```

Comment:
I wonder if this should be the default for CT. @cowboy maybe?

I also think `baseUrl` should be disallowed top level, it's E2E only. We should defs go over the config changes w/ a fine toothed comb to make sure everything is :100:, but this PR is definitely a good start.